pkg/bus/chat.go

```diff
@@ -36,6 +36,16 @@ func (b *Bus) chat(ctx context.Context, userID string, evt *pb.ChatMessage) erro
 		return err
 	}
 
+	// Regulate chat only if the user is not privileged and the
+	// chat is not a game chat
+	regulateChat := "regulated"
+	if sendingUser.IsAdmin ||
+		sendingUser.IsMod ||
+		sendingUser.IsDirector ||
```

Comment:
this may be counterintuitive but tournament/club directors are not actually directors in the database (rather, they don't have to be). I can fix this though by eventually assigning them director privileges.